apollo-router/src/uplink/license_stream.rs

```diff
@@ -196,29 +196,33 @@ fn reset_checks_for_licenses(
     if halt_at > now {
         // Only add halt if it isn't immediately going to be triggered.
         checks.insert_at(
-            Event::UpdateLicense(LicenseState::LicensedHalt { limits }),
+            Event::UpdateLicense(LicenseState::LicensedHalt {
+                limits: limits.clone(),
+            }),
```

Comment:
We are not adding the allowed features when creating the `LicenseLimits` above. Should we also be adding them when we do a license reset?